### DISCUSSION TITLE
Mod Compatibility? (Dramatic Doors, Sliding doors)

### DISCUSSION THREAD

**Discussion by Quickburst** [ThePandaOliver/Pandas-Extra-Details#26] [Ideas]
I absolutely 'a-door' (get it) this mod but have noticed that it reacts a little oddly to different door types from other mods. For example, Macaw's Doors works perfectly, however the sliding doors (such as the Shoji doors) do a weird hybrid of turning and sinking into the  wall. Another example is Dramatic Doors where the animation simply doesn't apply (despite them being basically regular doors but a block taller).

I think it'd be awesome to add some kind of compatibility or a way to apply animation to other blocks via a datapack.
Keep up the good work!

**Comment by ThePandaOliver** [comment]
I will try to add support for future updates.
Sorry for the late response, i was on vacation when you sent this.
